client/src/services/cryptpad.ts

```diff
@@ -43,9 +44,11 @@ export class Cryptpad {
   private containerElement: HTMLElement;
   private script: HTMLScriptElement;
   private instanceInitialized = false;
+  private informationManager: InformationManager;
 
-  constructor(containerElement: HTMLElement, serverUrl: string) {
+  constructor(containerElement: HTMLElement, serverUrl: string, informationManager: InformationManager) {
```

Comment:
This class should not take the information manager as a parameter. It should raise different types of error, and then whatever component is calling it should handle those errors and display the appropriate messages.